src/index.ts

```diff
@@ -1,26 +1,13 @@
-import 'reflect-metadata';
-import * as express from 'express';
-import * as bodyParser from 'body-parser';
-import * as helmet from 'helmet';
-import * as cors from 'cors';
-import * as morgan from 'morgan';
+require('dotenv-defaults').config();
```

Comment:
Please call ```config``` in a different line hopefully after import statements.
This pattern can cause a mess in other scenarios (more engs, more code).

Reply:
I know it looks and feels weird but this **needs** to be at the top. This line is actually **loading the values** defined in the `.env` files into `process.env`. We want to load all env vars before importing any other file in case they depend on them.

Reply:
we can also load the variables before even running the app/test process, so we don't need to worry about `dotenv`, to be honest, I'm fine with it for now but if there's an easy/maintainable way of replacing it, I would 🚀

Reply:
It's fine :) KISS for now.